hoomd/md/pytest/test_aniso_pair.py

```diff
@@ -282,8 +284,8 @@ def to_type_parameter_dicts(types, argument_dict):
                                    (0.01, 0.1, 2.0)],
                 "faces": [
                     md.pair.aniso.ALJ.get_ordered_vertices(vertices)[1]
-                    for vertices in shape_vertices
-                ]
+                    for vertices in shape_vertices[:-1]
+                ] + [[]]
```

Comment:
Why this change?

Reply:
This adds the vertices input for the ellipsoids test case, which needs to be an empty list.

Reply:
My bad on the previous comment, I meant to say that it gives the faces input. Passing an empty list into `get_ordered_vertices` causes an error, so I have to explicitly add the empty list here.

Reply:
I don't think the error can come from `get_ordered_vertices`, otherwise the code would fail without every constructing the list in the comprehension. Do you mean that `make_aniso_spec` fails, or maybe the simulation itself?

I realize this is a bit out of scope of what is currently changed in this PR, but in light of what we're testing I'm going to suggest a small pair of modifications to this last block modifying the `valid_params_list`.
1. Rather than wrapping all of the construction of `alj_arg_dict` in a try-except, can we just wrap the call to `get_ordered_vertices`? That makes it much clearer what triggers an error.
2. Rather than indiscriminately adding the extra empty set of faces, can we do that in an `if shape_vertices` block, again so that it's clear why this is needed?

Reply:
I'd rather address this in another small PR, this bugfix is important and needs to get merged so we can have it in the next beta release. Is everything in this PR other than this okay for merge?

Reply:
I can approve this PR so you can move forward, but I'd strongly recommend addressing this either in this PR or ASAP after it's merged. As the test is currently written it wasn't clear to reviewers that these changes to tests are linked to the bug fixed in this PR, which means any developer reworking these tests in the future could easily omit an equivalent test accidentally and allow bugs like to recur.

Reply:
Merging now as the release deadline is tomorrow. @tommy-waltmann will make a follow up PR to address this comment.